cypress/steps/yasqe-steps.ts

```diff
@@ -12,7 +12,7 @@ export class YasqeSteps {
     }
 
     static getExecuteQueryButton() {
-        return cy.get('.yasqe_queryButton');
+        return cy.get('.yasqe_queryButton:visible');
```

Comment:
Why is this needed?

Reply:
I had a problem with button selection when second tab is opened. I thought that selector finds the hidden button from first tab. I removed it now and test passed successfully, I will remove it.